iree/compiler/Conversion/LinalgToLLVM/KernelDispatch.cpp

```diff
@@ -104,6 +101,31 @@ llvm::SmallVector<int64_t, 4> getTileSizes(Operation *op) {
     }
   }
 
+  if (isa<linalg::GenericOp>(op)) {
+    auto linalgOp = dyn_cast<linalg::LinalgOp>(op);
+    switch (tilingLevel) {
+      case TilingLevel::WorkGroupTiles: {
+        llvm::SmallVector<int64_t, 4> workgroupTileSizes;
+        int iterationRank = linalgOp.iterator_types().size();
```

Comment:
Can you just do `iterationRank = getNumOuterParallelLoops(linalgOp);` this way you don't pick up the reduction dimensions? Also I'm guessing you only want to tile outer parallel loops to match what is done in flow?

Reply:
Right! I want to tile outer parallel loop and also skip inner reduction ones.